scalafix-reflect/src/main/scala/scalafix/reflect/RewriteInstrumentation.scala

```diff
@@ -21,6 +21,7 @@ object RewriteInstrumentation {
       def unapply(arg: Term): Boolean = arg match {
         case q"Rewrite.syntactic($_)" => true
         case q"Rewrite.semantic($_)" => true
+        case q"Rewrite.semantic($_)" => true
```

Comment:
Good catch! That was totally accidental, I was playing around with that line last night while prototyping on logo for scala.meta, it should never have been commited 😅 

Removed.